Mythri_projects/style.css

```diff
@@ -0,0 +1,114 @@
+html,body{
+    margin: 0;
+    width:100%;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 1rem;
+}
+.container{
+    margin:5px 30px;
+}
+
+
+ul li{
+    list-style:none;
+
+}
+
+img{
+    width:18%;
+    height: 10%;
+    padding:3px;
+}
+.nav ul{
+    margin-right: 250px;
+}
+
+ul li{
+    display: inline;
+    letter-spacing: 1.5px;
+    color: #eeeeee;
+}
+
+
+.navbar li a{
+    text-decoration:none;
+    color: #000a00;
+    letter-spacing: 1.5px;
+    margin-right: 30px;
+}
+.navbar{
+    margin-right: 300px;
+}
+
+.navbar,img{
+    display:inline-block;
+   
+}
+
+.innernav ul li span{
+    color: gold;
+    font-weight: 100;
+    font-size: small;
+}
+.innernav{
+    background-color: #111111;
+    padding:1px;
+    padding-right: 20px;
+    border-bottom: 3px solid gold;
+}
+.join{
+    float:right;
+    
+}
+
+.hero{
+    width:100%;
+    height:350px;
+}
+
+.container2{
+    margin:10px 50px;
+}
+
+h2{
+    font-weight: lighter;
+}
+
+h2 span{
+    color:red;
+    font-size:15px;
+    margin-left:10px;
+
+}
+.cardCont{
+    display:grid;
```

Comment:
for IE we need to use -ms-grid

Reply:
I have pushed a new file which contains the code for IE with -ms-grid